spring-cloud-gateway-webflux/pom.xml

```diff
@@ -26,8 +26,13 @@
 			<scope>test</scope>
 		</dependency>
 		<dependency>
-			<groupId>org.junit.vintage</groupId>
```

Comment:
Just removing this should be enough. Spring boot starter test brings in junit 5

Reply:
Done!